postcss.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = () => ( {
+	plugins: {
+		'postcss-custom-properties': {
+			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
+		},
```

Comment:
does this no longer need `preserve: preserveCssCustomProperties` to function in IE11?

Reply:
Ah, i see. We had the setting to allow folks to turn it off. Options that default true are weird. https://github.com/postcss/postcss-custom-properties#preserve

Reply:
Yes, `preserve` has the desired, IE11-friendly value, by default. We need to specify it only if we want to completely inline the CSS variable values and remove them from the CSS output. I'm not aware of any use case for that in Calypso.

Reply:
Here's some fun stuff related to `preserve`: https://github.com/Automattic/jetpack/pull/13854#issuecomment-550898168